### ISSUE TITLE
cli option --output-dir doesn't accept absolute path

### ISSUE THREAD

**Issue by blacksqr** [quarto-dev/quarto-cli#362]
If I execute the following commands:

```
quarto create-project --type website
quarto render --output-dir /tmp/quarto
```

the output files are not rendered to /tmp/quarto, but to the subdirctory tmp/quarto within the project directory.

**Comment by jjallaire** [comment]
That's correct, `output-dir` currently needs to be relative. This may change in the future, in the meantime I'd suggest just chaining the `render` call together with a call to `mv`

**Comment by akgold** [comment]
A note that this is particularly important if running inside a container. For example, I've built a container with a doc in it -- this is good for reproducibility. 

The `Dockerfile` closes with 
```
CMD quarto render . --output-dir /out
```

Then I would run this with e.g. `docker run -v $(pwd):/out` -- `/out` is **required** to be an absolute path by Docker. 

The `mv` after render works fine, but it was hard to debug, as I had to notice the lack of leading slash in the quarto output.
